awscli/examples/configure/_description.rst

```diff
@@ -34,4 +34,4 @@ The following configuration variables are supported in the config file:
 
 For more information on configuration options, see `Configuring the AWS Command Line Interface`_ in the *AWS CLI User Guide*.
 
-.. _`Configuring the AWS Command Line Interface`: http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html
```

Comment:
Can you revert this change? doesn't look like it's relevant here.

Reply:
Hi @kdaily, I'm not sure what GitHub is showing as being changed here. I tried putting the raw original text into two different diff checkers (https://www.diffchecker.com/ and https://text-compare.com/) and two different online editors that show hidden characters (https://www.soscisurvey.de/tools/view-chars.php and https://dostring.com/show-hidden-characters) and neither show a difference other than changing "you" to "your".

Reply:
Thanks for checking, I'm fine w/ leaving it then.